common/enablers/enableEpochsHandler.go

```diff
@@ -42,312 +47,636 @@ func NewEnableEpochsHandler(enableEpochsConfig config.EnableEpochs, epochNotifie
+		common.DoNotReturnOldBlockInBlockchainHookFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.DoNotReturnOldBlockInBlockchainHookEnableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.DoNotReturnOldBlockInBlockchainHookEnableEpoch,
+		},
+		common.AddFailedRelayedTxToInvalidMBsFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch < handler.enableEpochsConfig.AddFailedRelayedTxToInvalidMBsDisableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.AddFailedRelayedTxToInvalidMBsDisableEpoch,
+		},
+		common.SCRSizeInvariantOnBuiltInResultFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.SCRSizeInvariantOnBuiltInResultEnableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.SCRSizeInvariantOnBuiltInResultEnableEpoch,
+		},
+		common.CheckCorrectTokenIDForTransferRoleFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.CheckCorrectTokenIDForTransferRoleEnableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.CheckCorrectTokenIDForTransferRoleEnableEpoch,
+		},
+		common.FailExecutionOnEveryAPIErrorFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.FailExecutionOnEveryAPIErrorEnableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.FailExecutionOnEveryAPIErrorEnableEpoch,
+		},
+		common.MiniBlockPartialExecutionFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.MiniBlockPartialExecutionEnableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.MiniBlockPartialExecutionEnableEpoch,
+		},
+		common.ManagedCryptoAPIsFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.ManagedCryptoAPIsEnableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.ManagedCryptoAPIsEnableEpoch,
+		},
+		common.ESDTMetadataContinuousCleanupFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.ESDTMetadataContinuousCleanupEnableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.ESDTMetadataContinuousCleanupEnableEpoch,
+		},
+		common.FixAsyncCallbackCheckFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.ESDTMetadataContinuousCleanupEnableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.ESDTMetadataContinuousCleanupEnableEpoch,
+		},
+		common.SendAlwaysFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.ESDTMetadataContinuousCleanupEnableEpoch
+			},
+			activationEpoch: handler.enableEpochsConfig.ESDTMetadataContinuousCleanupEnableEpoch,
+		},
+		common.ChangeDelegationOwnerFlag: {
+			isActiveInEpoch: func(epoch uint32) bool {
+				return epoch >= handler.enableEpochsConfig.ESDTMetadataContinuousCleanupEnableEpoch
```

Comment:
is the flag correct?

Reply:
yes, [rc/v1.6.0 proof](https://github.com/multiversx/mx-chain-go/blob/rc/v1.6.0/common/enablers/enableEpochsHandler.go#L112)